### ISSUE TITLE
when i try to start ddlc with the mod in the correct folder (note: i had to re-install the game from steam) it gives me an error message instead from tracebacks.

### ISSUE THREAD

**Issue by WeirdLopster** [Monika-After-Story/MonikaModDev#8982]
this is the error message it gives me

I'm sorry, but an uncaught exception occurred.

While running game code:
ScriptError: Name (u'/home/potato/dddlc/0128/game/0config.rpy', 1644199079, 0) is defined twice, at game/0config.rpy:9 and Monika_After_Story-0.12.8-Mod/0config.rpyc:9.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\renpy\bootstrap.py", line 295, in bootstrap
    renpy.main.main()
  File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\renpy\main.py", line 364, in main
    renpy.game.script.load_script()  # sets renpy.game.script.
  File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\renpy\script.py", line 265, in load_script
    self.load_appropriate_file(".rpyc", ".rpy", dir, fn, initcode)
  File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\renpy\script.py", line 753, in load_appropriate_file
    self.finish_load(stmts, initcode, filename=fn + source)
  File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\renpy\script.py", line 419, in finish_load
    bad_node.filename, bad_node.linenumber))
ScriptError: Name (u'/home/potato/dddlc/0128/game/0config.rpy', 1644199079, 0) is defined twice, at game/0config.rpy:9 and Monika_After_Story-0.12.8-Mod/0config.rpyc:9.

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.8

**Comment by BrandiBlitz** [comment]
I'm having the same issue.

Also, icons end in ".ico"

![image](https://user-images.githubusercontent.com/101225938/165367075-ac368e3b-5c6a-4f00-a513-19c01c9cb066.jpeg)

**Comment by BrandiBlitz** [comment]
Update: dude, the extra stuff you left that is not related to yhe functionalitu of the mod (screenshots, junk code, etc.) may have been interfering with the mod. I got it working

The text boxes look like eye sores. I can't read it well. Traditional colours would have worked. Just saying.

![image](https://user-images.githubusercontent.com/101225938/165368310-54c29ed1-1fe7-41b8-bf8f-5ef5b0a630f3.jpeg)

**Comment by ThePotatoGuy** [comment]
@WeirdLopster  verify the steps [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-still-cant-run-the-game-and-none-of-the-above-faqs-solve-my-issue)

@BrandiBlitz seems you're using a mod that isn't Monika After Story? There is no menu like that in MAS.

**Comment by BrandiBlitz** [comment]
> @WeirdLopster  verify the steps [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-still-cant-run-the-game-and-none-of-the-above-faqs-solve-my-issue)
> 
> 
> 
> @BrandiBlitz seems you're using a mod that isn't Monika After Story? There is no menu like that in MAS. 

MAS uses the default textures alongside a "night" version of those default textures. 

Also, I found another issue. If you make it through to the option, then exit, the game breaks.
